### ISSUE TITLE
NTR: Anti-citrullinated protein antibody positivity

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5803]
**Preferred term label:**
Anti-citrullinated protein antibody positivity

**Synonyms**
ACPAs Positivity

**Definition (free text, please give PubMed ID)**
The presence autoantibodies in the serum that react do different citrullinated antigens, including filaggrin, fibrinogen, vimentin and collagen.

Comment: ACPAs are the major autoantibodies in rheumatoid arthritis and are highly specific. They are useful in the diagnosis and in the prediction oft he course and outcome of rheumatoid arthritis.

PMID: 27824549 ; doi: 10.5041/RMMJ.10255
PMID: 21713412 ; doi: 10.1007/s11926-011-0193-7

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
rheumatoid arthritis 

**Your nano-attribution (ORCID)**
lmu: chrapp, @katrinknoflach, @mgriese
